doc/threat_model/TBF_Headers.md

```diff
@@ -0,0 +1,16 @@
+TBF Headers
```

Comment:
You should define what TBF means somewhere in this document.

Reply:
Done.